jar-infer/jar-infer-lib/src/main/java/com/uber/nullaway/jarinfer/DefinitelyDerefedParams.java

```diff
@@ -58,9 +58,6 @@ private static void LOG(boolean cond, String tag, String msg) {
   private final ControlFlowGraph<SSAInstruction, ISSABasicBlock> cfg;
   private PrunedCFG<SSAInstruction, ISSABasicBlock> prunedCFG;
 
-  // used to resolve references to fields in putstatic instructions
-  private final IClassHierarchy cha;
```

Comment:
Why are we seeing changes to JarInfer and its WALA usage for this error prone change?

Reply:
It's an unused variable. Can put it back and suppress if need be?

Reply:
👍